sync2jira/downstream_issue.py

```diff
@@ -1152,6 +1152,26 @@ def sync_with_jira(issue, config):
         ):
             issue.content = pypandoc.convert_text(issue.content, "jira", format="gfm")
 
+    retry = False
```

Comment:
I think if I were writing this, I would have made `retry = False` into `retries = 0`, and each time you retry, increment the counter instead of setting it to `True`.

Then, the conditional looks like `if retries > 1: raise` or `if retries > 3: raise` to retry more aggressively.